examples/ShowRecorder.h

```diff
@@ -37,7 +37,8 @@
 class ShowRecorder {
  public:
   ShowRecorder(const std::string &filename,
-               const std::vector<unsigned int> &universes);
+               const std::vector<unsigned int> &universes,
+               const std::string &channeltrigger);
```

Comment:
It would be better to pass these values in as a native type, so the ShowRecorder class doesn't care about the format. Then if for example someone wrote a web based version of the recorder, they wouldn't have to convert the data into this random proprietary format to pass the config in.

See my discussion elsewhere and work out how many you may want to pass in, if it's one having two ints will make more sense, if it's more than one, having a hash or pairs may be more appropriate.